compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/InteropOrder.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.mpp.demo
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun InteropOrder() {
```

Comment:
I tried this strange example with `useInteropBlending = true` on Windows:
```
        Box {
            if (red) {
                Box(Modifier.size(150.dp).offset(0.dp, 0.dp).background(Color.Red))
                TestInteropView(Modifier.size(150.dp).offset(0.dp, 0.dp), Color.Red)
            }
            if (green) {
                TestInteropView(Modifier.size(150.dp).offset(75.dp, 75.dp), Color.Green)
            }
            if (blue) {
                Box(Modifier.size(150.dp).offset(150.dp, 150.dp).background(Color.Blue))
            }
        }
```

And sometimes it has a wrong ordering:
<img width="253" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/d37dff7a-cc29-4f81-9ba1-553eb6089e68">


Sometimes correct:
<img width="264" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/bbeda9fd-25f4-432d-bb56-71585d30f367">

Is it expected or is it a bug?

Reply:
A bug. Fixed by adding validate/repaint after adding of interop (this PR) + Fix from #1147